app/collections/tracks.js

```diff
@@ -1,24 +1,46 @@
 import Backbone from 'backbone';
 import Track from '../models/track';
-
+import cozysdk from 'cozysdk-client';
+import application from '../application'
 
 const Tracks = Backbone.Collection.extend({
+
     model: Track,
-    
-    comparator: function (collection) {
-        return collection.get('metas').title;
+
+    initialize(models, options) {
+        this.type = options.type;
+        if (this.type == 'upNext') {
+            this.listenTo(application, 'start', this.addCurrentToUpNext);
+        }
+    },
+
+    addCurrentToUpNext() {
+        this.listenTo(
+            application.appState,
+            'change:currentTrack',
+            function() {
+                let currentTrack = application.appState.get('currentTrack');
```

Comment:
The `currentTrack` value is passed here as `arguments[1]`. You can just declare your func as `function (appState, currentTrack) { ... }`.